manifests/cf-manifest/manifest/000-base-cf-deployment.yml

```diff
@@ -37,6 +37,11 @@ releases:
     version: 1.11.7
     url: https://github.com/cloudfoundry-community/nginx-release/releases/download/v1.11.7/nginx-1.11.7.tgz
     sha1: 133bb2260411b197924fff08d4cbd923cc8ec7eb
+    # FIXME: remove once CF is upgraded to > 276
+  - name: uaa
+    version: "52"
+    url: https://bosh.io/d/github.com/cloudfoundry/uaa-release?v=52
+    sha1: e7a1b9f368d30cc5e597be134844280b9887873e
```

Comment:
This change isn't sufficient. The [job definitions](https://github.com/alphagov/paas-cf/blob/master/manifests/cf-manifest/manifest/010-cf-jobs.yml#L123-L136) also need to be updated to point to this release, otherwise they'll continue to use uaa from cf-release.